test/e2e/testdata/load_balancing_consistent_hash_cookie.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: BackendTrafficPolicy
+metadata:
+  name: cookie-lb-policy
+  namespace: gateway-conformance-infra
+spec:
+  targetRefs:
+    - group: gateway.networking.k8s.io
+      kind: HTTPRoute
+      name: cookie-lb-route
+  loadBalancer:
+    type: ConsistentHash
+    consistentHash:
+      type: Cookie
+      cookie:
+        name: "Lb-Test"
```

Comment:
can we add an attribute `SameSite=Strict` in here and check in client side if it is being set ?